chapters/introduction.tex

```diff
@@ -30,94 +39,116 @@ \section{Doing credible research at scale}
+focusing on how to organize data work so that it is easy to code the desired analysis. 
+In the final chapter, we discuss publishing collaborative research- 
+both the research paper and the code and materials needed to recreate the results.  
+
+We will use broad terminology throughout this book to refer to research team members:
 \textbf{principal investigators (PIs)} who are responsible for
-the overall success of the project;
+the overall design and stewardship of the study;
 \textbf{field coordinators (FCs)} who are responsible for
-the operation of the project on the ground;
+the implementation of the study on the ground;
 and \textbf{research assistants (RAs)} who are responsible for
-handling technical capacity and analytical tasks.
+handling raw data processing and analytical tasks.
+
+
+\section{Adopting reproducible workflows}
+We will provide free, open-source, and platform-agnostic tools wherever possible,
+and point to more detailed instructions when relevant.
+Stata is the notable exception here due to its current popularity in economics.
+Most tools have a learning and adaptation process,
+meaning you will become most comfortable with each tool
+only by using it in real-world work.
+Get to know them well early on,
+so that you do not spend a lot of time learning through trial and error.
+
+While adopting the workflows and mindsets described in this book requires an up-front cost,
+it will save you (and your collaborators) a lot of time and hassle very quickly.
+In part this is because you will learn how to implement essential practices directly;
+in part because you will find tools for the more advanced practices;
+and most importantly because you will acquire the mindset of doing research with a high-quality data focus.
+We hope you will find this book helpful for accomplishing all of the above,
+and that mastery of data helps you make an impact!
+
 
 \section{Writing reproducible code in a collaborative environment}
+Throughout the book, we refer to the importance of good coding practices. 
+These are the foundation of reproducible and credible data work,
+and a core part of the new data science of development research.
+Code today is no longer a means to an end (such as a research paper),
+rather it is part of the output itself: a means for communicating how something was done,
+in a world where the credibility and transparency of data cleaning and analysis is increasingly important.
+As this is fundamental to the remainder of the book's content, 
+we provide here a brief introduction to ``good'' code and standardized practices.
+
+``Good'' code has two elements:
+- it is correct (doesn't produce any errors along the way)
+- it is useful and comprehensible to someone who hasn't seen it before (including the author three weeks later)
+Many researchers have been trained to code correctly. 
+However, when your code runs on your computer and you get the correct results, you are only half-done writing \underline{good} code.
```

Comment:
```suggestion
However, when your code runs on your computer and you get the correct results, you are only half-done writing \textit{good} code.
```

For some reason, I do not know why, we do not use underlining anywhere else in the book, so it stood out to me. And I did not think the underlining looked very well in our format definition. I rather have us italicize here.